examples/node-groups/managed-node-groups/main.tf

```diff
@@ -260,6 +270,32 @@ module "eks_blueprints" {
         subnet_type = "private"
       }
     }
+    # Managed Node group with Launch templates using AMI TYPE and SPOT instances of 2 vCPUs and 8 Gib Memory
+    spot_2vcpu_8mem = {
+      node_group_name = "mng-spot-2vcpu-8mem"
+      capacity_type   = "SPOT"
+      instance_types  = ["m5.large", "m4.large", "m6a.large", "m5a.large", "m5d.large"]
```

Comment:
we can't mix x86 and AMD instance types - the AMI pulled only supports one or the other

Reply:
so, no change here, right? there's no ARM instance type in the list